common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +33,13 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            for key in event.iterkeys():
+                if isinstance(event[key], str):
+                    event[key] = event[key].decode('latin1')
```

Comment:
I understand what is described in the ticket, and I understand that this is dealing with the symptoms of what was observed in the ticket.  But this doesn't actually fix the problem, whatever it is, and I was hoping that there might be some inquiry into how those characters got there.  And although I reported observing a few latin1 characters, has there been any survey of the errors to know that they are all latin1?  And even if it were latin1, one could use utf8 as well....